numpy/core/src/umath/ufunc_object.c

```diff
@@ -5579,7 +5579,7 @@ PyUFunc_RegisterLoopForType(PyUFuncObject *ufunc,
     cobj = PyDict_GetItemWithError(ufunc->userloops, key);
     if (cobj == NULL && PyErr_Occurred()) {
         Py_DECREF(key);
```

Comment:
```suggestion
```
Thanks, looks good, except that key is already being decref'd by the fail cleanup.

Reply:
Whoops, fixed.